sirepo/template/madx.py

```diff
@@ -270,10 +271,69 @@ def _extract_report_data(data, run_dir):
     )(data, run_dir)
 
 
+def _extract_report_bunchReport(data, run_dir):
+    util = LatticeUtil(data, _SCHEMA)
+    m = util.find_first_command(data, 'twiss')
+    if not m:
```

Comment:
Twiss command is in default-data and can't be deleted in the GUI if it is the last one so it is required. I don't think we need to check.